server/src/main/java/hu/flowacademy/epsilon/sport_event_organizer/email/MailService.java

```diff
@@ -0,0 +1,178 @@
+package hu.flowacademy.epsilon.sport_event_organizer.email;
+
+import hu.flowacademy.epsilon.sport_event_organizer.model.Cup;
+import hu.flowacademy.epsilon.sport_event_organizer.model.Team;
+import hu.flowacademy.epsilon.sport_event_organizer.model.User;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.scheduling.annotation.EnableAsync;
+import org.springframework.stereotype.Component;
+
+import java.util.Properties;
+
+import javax.mail.Message;
+import javax.mail.MessagingException;
+import javax.mail.PasswordAuthentication;
+import javax.mail.Session;
+import javax.mail.Transport;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+
+
+@Component
+@EnableAsync
+public class MailService {
+
+
+    @Value("${app.url}")
+    private String homePageUrl;
+
+    final String username = "projektmunkasports@gmail.com";
```

Comment:
Legyen private és jöjjön az értéke az application.yml-ből, mint a homePageUrl-nek. Ugyan ez a password is